ckanext/scheming/templates/scheming/organization/group_form.html

```diff
@@ -34,13 +35,13 @@
             {% block save_text %}
               {%- if action == "edit" -%}
                 {%- if 'humanize_entity_type' in h -%}
-                  {{ h.humanize_entity_type('organization', group_type, 'update label') }}
+                  {{ h.humanize_entity_type('Organization', group_type, 'update label') }}
```

Comment:
I don't think this is needed

Reply:
the button is None... because of this check [here](https://github.com/ckan/ckan/blob/34401f000846aa59559c77e755e56656cf1355f9/ckan/lib/helpers.py#L1036)

Reply:
Then the problem is that `group_type` doesn't match `organization`. In any case that's a separate issue, so please create a separate PR